### PR TITLE
adding flag to pomodoroDto

### PR DESCRIPTION
**Pull request by dejanjesic**
true if pomodoro has been paused, otherwise it won't mapped

### CODE REVIEW

File: service/src/main/java/eu/execom/labs/pomodorospringreact/dto/PomodoroDto.java [hunk 1]
```diff
@@ -24,6 +25,8 @@
 
     private Date endTime;
 
+    private Date pauseStartTime;
```
Comment (stanislavgrujic) [inline service/src/main/java/eu/execom/labs/pomodorospringreact/dto/PomodoroDto.java]: For what is this field used?

Comment (dejanjesic) [reply service/src/main/java/eu/execom/labs/pomodorospringreact/dto/PomodoroDto.java]: If user stops pomodoro and close tab or browser soon after that, client should know when pomodoro was paused if he wants to resume it later in order to calculate pomodoro's new end time.

Comment (stanislavgrujic) [reply service/src/main/java/eu/execom/labs/pomodorospringreact/dto/PomodoroDto.java]: That is ok, but I don't see any logic doing what you just described... The idea is that you also make the change together with the addition of the field.

Comment (dejanjesic) [reply service/src/main/java/eu/execom/labs/pomodorospringreact/dto/PomodoroDto.java]: That was done in my previous PR you just merged, we did some logic on our side but we wasn't sending information about pauseStartTime to the client and now we do.

**Pull request merged**